src/FlaUI.WebDriver/Keys.cs

```diff
@@ -0,0 +1,505 @@
+﻿using FlaUI.Core.WindowsAPI;
+
+namespace FlaUI.WebDriver;
+
+internal class Keys
+{
+    /// <summary>
+    /// Normalized key mapping from https://www.w3.org/TR/webdriver2/#keyboard-actions
+    /// </summary>
+    private static readonly Dictionary<char, string> s_normalizedKeys = new Dictionary<char, string>()
+    {
+        { '\uE000', "Unidentified" },
```

Comment:
The unicode chars in this list should maybe use the constants defined below, but that makes checking that they're correct a bit more difficult as it involves a double-lookup for each.